src/libs/highlevel/README.md

```diff
@@ -166,32 +164,32 @@ The default callback simply logs the error with `ELEKTRA_LOG_DEBUG` and then cal
 
 TODO: why DEBUG? It should be ERROR? And we should enable logging by default, otherwise users will not see the message?
```

Comment:
I couldn't add a comment in line 145:
```
TODO: explanation of low-level code or removal of this feature for this release?
```

What is that supposed to mean?

Reply:
low-level error code APIs are imho not properly explained. You have two options:

1. remove the APIs
2. properly explain them